docs/how-to/add-model-and-api-endpoints.md

```diff
@@ -99,6 +116,46 @@ class RecurringEvent(AbstractBaseModel):  # (1)!
 1. Try to add the relationships to non-existent models, but comment them out. Another developer will complete them when they go to implement those models.
 1. Always override the `__str__` function to output something more meaningful than the default. It lets us do a quick test of the model by calling `str([model])`. It's also useful for the admin site model list view.
 
+??? note "Updating models.py for many-to-many relationships"
+    For adding many-to-many relationships with additional fields, such as `ended_on`, we can add
+
+    ```python title="app/core/tests/test_models.py" linenums="1"
```

Comment:
I believe you meant `core/models.py` instead of `app/core/tests/test_models.py`.